test/PowerShellEditorServices.Test/Session/ScriptFileTests.cs

```diff
@@ -606,9 +611,9 @@ public void DocumentUriRetunsCorrectStringForAbsolutePath()
                 scriptFile = new ScriptFile(path, path, emptyStringReader, PowerShellVersion);
                 Assert.Equal("file:///home/AlexKamal/projects/Rocinate/ProtoMolecule.ps1", scriptFile.DocumentUri);
 
-                path = "/home/BobbyDraper/projects/Rocinate/foo's_~#-[@] +,;=%.ps1";
+                path = "/home/BobbieDraper/projects/Rocinate/foo's_~#-[@] +,;=%.ps1";
```

Comment:
:rofl:

Reply:
This kind of thing happens when you listen to Audible books (instead of reading them). :-)